app/components/rdf-input-fields/geboorte-datum-input.js

```diff
@@ -0,0 +1,108 @@
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+import InputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/input-field';
+import { triplesForPath } from '@lblod/submission-form-helpers';
+import { replaceSingleFormValue } from 'frontend-lmb/utils/replaceSingleFormValue';
+import moment from 'moment';
+import { NamedNode } from 'rdflib';
+
+export default class RDFGeboorteDatumInput extends InputFieldComponent {
+  inputId = 'birthdate-' + guidFor(this);
+
+  @service store;
+  @tracked geboortedatum;
+
+  minDate;
+  maxDate;
+
+  constructor() {
+    super(...arguments);
+
+    const now = new Date();
+    const hundredYearsAgo = new Date(
+      now.getFullYear() - 100,
+      now.getMonth(),
+      now.getDate()
+    );
+    const eighteenYearsAgo = new Date(
+      now.getFullYear() - 18,
```

Comment:
Maybe specify that 18 is the age of majority in Belgium?

Reply:
Keep this for now but i wonder if this is a correct requirement. Will check with Tim. (what happens if we are dealing with a jeugdraad or something?)

Reply:
I think validation is something that should reviewed for all forms, so at that stage these requirements can also be revisited, and this works as a proof of concept for how to add custom validations for now.